test/axes/numericAxisTests.ts

```diff
@@ -431,4 +431,36 @@ describe("NumericAxis", () => {
     svg.remove();
   });
 
+  it("reasonably approximates tick label sizes with approximate measuring", () => {
+    var SVG_WIDTH = 500;
+    var SVG_HEIGHT = 100;
+
+    [[-1, 1],
+     [0, 10],
+     [0, 999999999]
+    ].forEach((domainBounds) => {
+      var scale = new Plottable.Scales.Linear();
+      scale.domain(domainBounds);
+
+      var svgApprox = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+      var numericAxisApprox = new Plottable.Axes.Numeric(scale, "left");
+      numericAxisApprox.usesTextWidthApproximation(true);
+      numericAxisApprox.renderTo(svgApprox);
+      var widthApprox = (<any> numericAxisApprox)._computeWidth();
+
+      var svgExact = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
```

Comment:
Through the rest of our tests, we usually (I think actually always) use one svg to do everything in.  Mainly to keep with consistency in usability and codewise, can you just use one svg?

Maybe like
First, render using no approximation and get the width.
Second, set the flag to use approximation and get the width.
Compare the widths.